src/test/java/core/basesyntax/strategy/SupplyOperationTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class SupplyOperationTest {
+    private FruitOperationHandler supplyOperation;
```

Comment:
You should supply the initial inventory map to the `SupplyOperation` constructor in `setUp` method. It would make your tests more isolated and reliable.